### PR TITLE
Dashboard: Blog  and docs

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @lickem22 
Estimate: 10mins

---

## Ticket

Fixes: [#606](https://idinsight.atlassian.net/browse/AAQ-606)
Fixes: [#537](https://idinsight.atlassian.net/browse/AAQ-537)

### Goal

Add blog and docs for new dashboard

## How has this been tested?

Locally using mkdocs

## Checklist

Fill with `x` for completed. 

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts

(Delete any items below that are not relevant)
- [ ] I have updated the automated tests
- [ ] I have updated the scripts in `scripts/`
- [ ] I have updated the requirements
- [ ] I have updated the README file
- [x] I have updated affected documentation
- [x] I have added a blogpost in Latest Updates
- [ ] I have updated the CI/CD scripts in `.github/workflows/`
- [ ] I have updated the Terraform code

### CODE REVIEW

File: docs/blog/posts/dashboard-overview-page.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - Admin App
+  - Dashboard
+date: 2024-07-10
+---
+
+# Revamping the dashboard
+
+In the last month or so, we've had support from Google fellows to pinch hit on a few items.
+Ahn Mac, a UI/UX designer at Google, helped design a new dashboard that we are very excited about.
+The first page, "Overview" is now ready for you to check out.
```
Comment (tonyzhao6) [inline docs/blog/posts/dashboard-overview-page.md:14]: Nit: "The first page, "Overview," is...

**Review by tonyzhao6 — COMMENTED** [review]
LGTM

File: docs/blog/posts/content-dashboard.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ authors:
   - Carlos
```
Comment (lickem22) [inline docs/blog/posts/content-dashboard.md:3]: You should probably update yourself as the author.

File: docs/components/admin-app/index.md [hunk 3]
```diff
@@ -34,6 +34,8 @@
     Allows you to see statistices like which content is most frequently being
```
Comment (lickem22) [inline docs/components/admin-app/index.md:34]:
```suggestion
    Allows you to see statistics like which content is most frequently being
```

**Review by lickem22 — APPROVED** [review]
Thanks Sid. One typo to fix.

**Pull request merged**